CHANGELOG.md

```diff
@@ -5203,6 +5203,8 @@ podspec, then update these method calls.
 
 * Updated Xcodeproj to have a friendlier API.
 
+* Created `NOMENCLATURE.md` to keep a glossary of the most common terms used in cocoapods.
```

Comment:
Can this be moved near the top of the CHANGELOG.md? There's a section near the top called master. You can add it to the enhancements section there, and then you can copy the formatting used in previous enhancements sections.